docs/topics/clusterdefinitions.md

```diff
@@ -65,6 +65,7 @@ $ aks-engine get-versions
 | gcLowThreshold                   | no                        | Sets the --image-gc-low-threshold value on the kublet configuration. Default is 80. [See kubelet Garbage Collection](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | kubeletConfig                    | no                        | Configure various runtime configuration for kubelet. See `kubeletConfig` [below](#feat-kubelet-config)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| kubeReservedCgroup               | no                        | The name of a systemd slice to create for containtment of both kubelet and the container runtime. This should not point to an existing systemd slice. Defaults to the empty string, which means kubelet and the container runtime will be placed in the system slice.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I think it's more accurate to say:

> Defaults to "kubereserved", which means kubelet and the container runtime will be placed in the system slice.

Is that right?

Reply:
No, I think the current phrasing is correct. The default behavior is always to put them in the system slice unless specified. If you specify anything else, that slice will be created and the units will be placed inside it.

Will update for clarity

Reply:
I think the "Defaults to the empty string" is what's confusing to me, as the default of that property will actually be "kubereserved".